test/spec/services/applicationGeneratorSpec.js

```diff
@@ -255,77 +255,136 @@ describe("ApplicationGenerator", function(){
     });
 
     it("should generate a BuildConfig for the source", function(){
-      expect(resources.buildConfig).toEqual(
+      var generatedBuildConfig = resources.buildConfig;
+
+      // specific tests to ease pain in debugging
```

Comment:
In the long run, these are going to make the tests harder to maintain. We've repeated everything twice, and it's a lot of work to write these. I'm not sure it's worth the tradeoff.

Reply:
@benjaminapetersen I'd rather than test the same things twice. Since we have the diff reporter, let's remove this.

Reply:
The diff reporter states this is legacy behavior.  Boo.  I'm working on getting more info from the developer.  It may mean that he isn't supporting it anymore.  Mind if I hold off on deleting the code till I have a good answer? I hate to update all our dependencies just to end up with a deprecated dependency that will fall over on us again.  Hopefully will get an answer soon.